docs/content/preview/explore/query-1-performance/explain-analyze.md

```diff
@@ -2,455 +2,274 @@
-    AND contacts.is_over_charged = TRUE
-    AND (updated_at > '2021-04-12 12:00:00');
-```
-
-```output
-QUERY PLAN
-----------------------------------------------
-Aggregate (cost=107.50..107.51 rows=1 width=8)
-    Seq Scan on contacts (cost=0.00..105.00 rows=1000 width=0)
-        Filter: (is_paid AND is_over_charged AND (updated_at > '2021-04-12 12:00:00'::timestamp without time zone) AND (account_id = 5678))
-(3 rows)
+CREATE INDEX idx_value_1 ON kvstore(value);
 ```
 
-```sql
-yugabyte=# explain SELECT COUNT(*) FROM contacts WHERE contacts.account_id = 7890
-    AND contacts.is_paid = FALSE
-    AND (updated_at > '2021-04-12 12:00:00');
-```
+Now if you run the same fetch by value query, you should see an output similar to,
 
-```output
-QUERY PLAN
-----------------------------------------------
-Aggregate (cost=107.50..107.51 rows=1 width=8)
-    Seq Scan on contacts (cost=0.00..105.00 rows=1000 width=0)
-        Filter: ((NOT is_paid) AND (updated_at > '2021-04-12 12:00:00'::timestamp without time zone) AND (account_id =1234))
-(3 rows)
+```yaml{linenos=inline, .nocopy}
+# EXPLAIN (ANALYZE, DIST, COSTS OFF) select * from kvstore where value='85d083991d';
+                                    QUERY PLAN
+-----------------------------------------------------------------------------------
+ Index Scan using idx_value_1 on kvstore (actual time=1.884..1.888 rows=1 loops=1)
+   Index Cond: ((value)::text = '85d083991d'::text)
+   Storage Table Read Requests: 1
+   Storage Table Read Execution Time: 0.861 ms
+   Storage Table Rows Scanned: 1
+   Storage Index Read Requests: 1
+   Storage Index Read Execution Time: 0.867 ms
+   Storage Index Rows Scanned: 1
+ Planning Time: 0.063 ms
+ Execution Time: 1.934 ms
+ Storage Read Requests: 2
+ Storage Read Execution Time: 1.729 ms
+ Storage Rows Scanned: 2
+ Storage Execution Time: 1.729 ms
+ Peak Memory Usage: 24 kB
 ```
 
-In each case, the queries do a `Seq Scan` (sequential scan) on the tables. This operation requires scanning the entire table to retrieve the desired columns. Even using the partition keys to do the lookup, it still needs to do a lot of scanning.
-
-Avoid `SELECT COUNT(*)` queries in most cases, as they can require a full scan of the table to get the results. This can cause query degradation, and in some cases cause the query to not return at all.
+In this run, Index scan was used and the execution time has been lowered (`1.934 ms` - 40% improvement) . Only one row was returned (via `rows=1`), but two rows were scanned (`Storage Rows Scanned: 2`) as one row was looked up from the index (via `Storage Index Rows Scanned: 1`) and one row was looked up from the table (via `Storage Table Rows Scanned: 1`). This is because, the executor looked up the row for the matching value: `85d083991d` from the index but had to go to the main table to fetch the key, sa we have specified `SELECT *` in our query. Only the values are present in the index and not the keys.
```

Comment:
```suggestion
In this run, Index scan was used and the execution time has been lowered (`1.934 ms` - 40% improvement). Only one row was returned (via `rows=1`), but two rows were scanned (`Storage Rows Scanned: 2`) as one row was looked up from the index (via `Storage Index Rows Scanned: 1`) and one row was looked up from the table (via `Storage Table Rows Scanned: 1`). This is because, the executor looked up the row for the matching value: `85d083991d` from the index but had to go to the main table to fetch the key, as we have specified `SELECT *` in our query. Only the values are present in the index and not the keys.
```